code/datums/components/crafting/tailoring.dm

```diff
@@ -50,6 +50,24 @@
 	time = 20
 	category = CAT_CLOTHING
 
+/datum/crafting_recipe/hudsunciv
+	name = "Civilian HUDsunglasses"
+	result = /obj/item/clothing/glasses/hud/civ/sunglasses
+	time = 20
+	tool_behaviors = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	reqs = list(/obj/item/clothing/glasses/hud/civ = 1,
```

Comment:
format lists like this
list(
	thing1,
	thing2,
)

Reply:
Again, this is a slightly altered copy/paste of code elsewhere in this same file. I could change it, but at that point I might as well change the rest of them for consistency and then I'm cleaning up lots of things.

Reply:
ok then do that

Reply:
you dont need to change it everywhere but its definitely good practice to change it here